### PR TITLE
beta version

### PR DESCRIPTION
**Pull request by Anton-Zhytonbaiev**
- [DEMO LINK](https://Anton-Zhytonbaiev.github.io/layout_catalog/)
- [TEST REPORT LINK](https://Anton-Zhytonbaiev.github.io/layout_catalog/report/html_report/)

### CODE REVIEW

File: src/index.html [hunk 1]
```diff
@@ -5,8 +5,437 @@
...
+          </li>
+          <li class="header__item">
+            <a class="header__link" href="#video">video</a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+
+    <main>
+      <div class="cards">
```
Comment (vanst1) [inline src/index.html:68]: move class="cards" to the main tag

File: src/styles/main.scss [hunk 2]
```diff
@@ -1,3 +1,20 @@
...
+  font-size: 12px;
+}
+
 body {
   margin: 0;
 }
+
+p {
+  margin: 0;
+}
```
Comment (vanst1) [inline src/styles/main.scss:13]: create `_reset.scss` file for this

File: src/styles/main.scss [hunk 3]
```diff
@@ -1,3 +1,20 @@
+html {
+  font-family: "Roboto", sans-serif;
+  font-weight: 500;
+  font-size: 12px;
+}
```
Comment (vanst1) [inline src/styles/main.scss:5]: create `_normalize.scss` file for this

File: src/styles/blocks/cards.scss [hunk 4]
```diff
@@ -0,0 +1,17 @@
...
+}
+
+.cards__card {
+  margin-right: 48px;
+  margin-bottom: 46px;
+}
+
+.cards__card:nth-child(4n) {
+  margin-right: 0;
+}
```
Comment (vanst1) [inline src/styles/blocks/cards.scss:17]: checklist 10, use `&` :

```
.cards {
  display: flex;
  flex-wrap: wrap;
  justify-content: center;
  margin: 0 auto;
  padding-top: 50px;
  width: getCountWidth($width-card, $space);


  &__card {
  margin-right: 48px;
  margin-bottom: 46px;
}

  &__card:nth-child(4n) {
  margin-right: 0;
}
}
```

File: src/styles/blocks/header.scss [hunk 5]
```diff
@@ -0,0 +1,67 @@
+ul {
+  margin: 0;
+  padding: 0;
+}
+
```
Comment (vanst1) [inline src/styles/blocks/header.scss:5]: create `_reset.scss` file for this

File: src/styles/blocks/header.scss [hunk 6]
```diff
@@ -0,0 +1,67 @@
...
+li {
+  text-transform: uppercase;
+  list-style: none;
+}
```
Comment (vanst1) [inline src/styles/blocks/header.scss:9]: create `_normalize.scss` file for this

**Review by vanst1 — APPROVED** [review]
Good job:)
Check comments)

File: src/index.html [hunk 7]
```diff
@@ -5,8 +5,437 @@
...
+      <a href="#home">
+        <img
+          class="header__logo"
+          src="images/logo.png"
+          alt="MOYO"
+        >
+      </a>
+      <nav>
+        <ul class="header__list">
+          <li class="header__item">
```
Comment (Denchepornyuk) [inline src/index.html:23]: nav should be a separate bem block, create `nav` block and elements from this block

File: src/index.html (hunk 1 above)
Comment (Denchepornyuk) [inline src/index.html:68]:
```suggestion
      <div class="catalog">
```

File: src/index.html [hunk 8]
```diff
@@ -5,8 +5,437 @@
...
+            src="images/imac.jpeg"
+            alt="APPLE A1419 iMac 27"
+          >
+          <h1 class="card__name name">
+            APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+          </h1>
+
+          <p class="card__code code">
+              Product code: 195434
+          </p>
```
Comment (Denchepornyuk) [inline src/index.html:81]: do you need `name` and `code` separated classes?

File: src/styles/blocks/card.scss [hunk 9]
```diff
@@ -0,0 +1,105 @@
...
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  width: 200px;
+}
+
+.card__image {
+  width: 160px;
+  height: 134px;
+  margin: 32px auto 40px;
+}
```
Comment (Denchepornyuk) [inline src/styles/blocks/card.scss:15]: you scss nesting with `&`

File: src/styles/blocks/card.scss [hunk 10]
```diff
@@ -0,0 +1,105 @@
...
+  color: #060b35;
+  display: flex;
+  width: 166px;
+  height: 36px;
+  line-height: 18px;
+}
+
+.name {
+  margin: 0 auto 4px;
+}
```
Comment (Denchepornyuk) [inline src/styles/blocks/card.scss:29]: remove class `name` and move this code to  `card__name`
you can add margins for bem elements, you can't add margins only for bem blocks

File: src/styles/blocks/card.scss [hunk 11]
```diff
@@ -0,0 +1,105 @@
...
+  line-height: 14px;
+  color: #616070;
+  width: 98px;
+  height: 14px;
+  display: flex;
+}
+
+.code {
+  margin-left: 16px;
+}
```
Comment (Denchepornyuk) [inline src/styles/blocks/card.scss:43]: each bem block should be in separate file

File: src/styles/blocks/header.scss (hunk 5 above)
Comment (Denchepornyuk) [reply src/styles/blocks/header.scss:5]: agree, let's create _reset file

**Review by Denchepornyuk — CHANGES_REQUESTED** [review]

File: src/index.html [hunk 12]
```diff
@@ -5,8 +5,437 @@
...
-    <h1>Catalog</h1>
+    <header class="header">
+      <a href="#home">
+        <img
+          class="header__logo"
+          src="images/logo.png"
+          alt="MOYO"
+        >
+      </a>
+      <nav>
```
Comment (Denchepornyuk) [inline src/index.html:21]: nav should be a separate bem block

**Review by Denchepornyuk — CHANGES_REQUESTED** [review]

**Review by Denchepornyuk — APPROVED** [review]

**Pull request closed without merging**